src/aiohomeconnect/model/error.py

```diff
@@ -101,3 +101,11 @@ class InternalServerError(HomeConnectError):
 @dataclass
 class Conflict(HomeConnectError):  # noqa: N818
     """Represent Conflict."""
+
+
+class HomeConnectRequestError(Exception):
```

Comment:
We should have a base exception for the library that all exceptions inherit. If rather have that named `HomeConnectError` and rename the exception that has the model description and key.